frontend/language/src/nb.json

```
@@ -20,6 +20,7 @@
   "api_errors.DM_05": "Kunne ikke lese ugyldig XML.",
   "api_errors.DM_CsharpCompiler_NameCollision": "Navnet <bold>{{nodeName}}</bold> er i bruk på et element som har et overordnet element med samme navn. Gi et av disse elementene et annet navn.",
   "api_errors.GT_01": "Det er en konflikt i applikasjonen",
+  "api_errors.GT_03": "Sesjonen er utløpt. Du blir nå logget ut. Prøv på nytt.",
```

Gørild [suggested](https://github.com/Altinn/altinn-studio/issues/12582#issuecomment-2164754266) a new text:
```suggestion
  "api_errors.GT_03": "Du blir logget ut fordi økten din har utløpt. Logg inn på nytt.",
```

---

fixed